bin/build_pgms.sh

```diff
@@ -1,7 +1,2 @@
 #!/bin/sh
-echo '"Use -m32 switch to force 32-bit build"'
-${CXX:-g++} $* -std=c++17 -Wall -DNDEBUG -O2 -o bin/lists src/date.cpp src/issues.cpp src/status.cpp src/sections.cpp src/mailing_info.cpp src/report_generator.cpp src/lists.cpp
-${CXX:-g++} $* -std=c++17 -Wall -o bin/section_data src/section_data.cpp
-${CXX:-g++} $* -std=c++17 -Wall -o bin/toc_diff src/toc_diff.cpp
-${CXX:-g++} $* -std=c++17 -Wall -DNDEBUG -O2 -o bin/list_issues src/date.cpp src/issues.cpp src/status.cpp src/sections.cpp src/list_issues.cpp
-${CXX:-g++} $* -std=c++17 -Wall -DNDEBUG -O2 -o bin/set_status  src/set_status.cpp src/status.cpp
+make -C $(dirname $0)/.. pgms
```

Comment:
I don't understand the point of this shell script, it's strictly inferior to the makefile as it doesn't reuse objects that are used in multiple programs.
I suppose to be equivalent to the original, maybe it should `make clean` first, so that it always rebuilds everything (but would still reuse objects for multiple programs).

Reply:
```suggestion
make -C $(dirname $0)/.. clean pgms
```

Reply:
One difference is that this script uses `-DNDEBUG` and the makefile doesn't (it leaves `assert` enabled _and_ turns on libstdc++ assertions). But the performance is more than acceptable even with assertions enabled.

Reply:
Ah it also builds without `-g` which makes it a few seconds faster than `make pgms` in the makefile, despite compiling some source files more than once.

So this script is for building an optimized, un-debuggable "production" build ... but do we need that?

Running `make lists` takes about 2s for the assertion-enabled binaries built by the makefile, and about 2s for the assertion-disabled binaries built by this script (and of course `-g` makes no difference to runtime performance).

I see no reason for this script to do anything different from the makefile. Maybe we should just remove it.

The how-to-docs.html file could also be greatly simplified. `git clone ... && cd issues && make lists` is all you need now, generating the gh-pages is done after pushing to github now.

Reply:
> The how-to-docs.html file could also be greatly simplified. `git clone ... && cd issues && make lists` is all you need now, generating the gh-pages is done after pushing to github now.

See #441

Reply:
I'm sorry that I caused a merge conflict here. Feel free to revert my PULL request and merge your suggestion instead.

Reply:
No problem, I've rebased this change and now it just replaces the `.sh` script.

Does this work for you? I assume you're using the `build_pgms.bat` file instead?